src/ucs/arch/x86_64/cpu.c

```diff
@@ -324,7 +324,7 @@ ucs_cpu_model_t ucs_arch_get_cpu_model()
     if (family == 0xf) {
         family += version.ext_family;
     }
-    if (family == 0x6 || family == 0xf) {
+    if (family == 0x6 || family == 0xf || family == 0x17) {
```

Comment:
could you wrap all conditions by `()`, please?
it was missed even before your changes, but it has to be following [UCX Code Style](https://github.com/openucx/ucx/blob/master/doc/CodeStyle.md)
thank you!

Reply:
sure